WcaOnRails/app/models/locale.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+class Locale < SimpleDelegator
+  SOME_CHARS = '[\s\S]*?'
+  COMMENT_LINE_GROUP = '((?:\s*#.*\n)*)'
+  KEY_MATCHER = "['\"]?%s['\"]?:"
+  HASHTAG = "hash: "
+
+  attr_accessor :locale
+
+  def initialize(locale, is_translation=false)
+    self.locale = locale.to_s
+    filename = Rails.root.join('config', 'locales_test', "#{locale}.yml")
```

Comment:
This is actually only for debug purpose: I intend to remove the `locales_test` directory and directly load the real locale :)
I just wanted to mimic the real behavior with "fake" locales.